### ISSUE TITLE
SSD wear level not measured

### ISSUE THREAD

**Issue by Bullion-Manor** [openhardwaremonitor/openhardwaremonitor#1398]
I use Windows 10. My device has an SSD for storage, but this program detects it as a "Generic Hard Disk". It measured my storage usage correctly within what seems to be 2% error according to the data from Disk Management. My question now is if SSDs are supposed to be detected as a "General Hard Disk", and whether or not this is an actual bug. Regardless, an issue persists because the [official website's documentation page](https://openhardwaremonitor.org/documentation/) said that SSD wear level is supposed to be displayed, but it isn't.

[OpenHardwareMonitor.Report.txt](https://github.com/openhardwaremonitor/openhardwaremonitor/files/5941368/OpenHardwareMonitor.Report.txt)

**Comment by evg656e** [comment]
Looks like it does not recognize NVMe drives
![image](https://user-images.githubusercontent.com/28028005/107751679-ca9ad500-6d2e-11eb-9b37-b0014d9db597.png)

**Comment by donpin1** [comment]
Agree, please add NVMe drives and their temperatures.  It would be great if you also had a tab that monitored SMART attributes.   Nevertheless, this is easily the best program for me that I have come across. Well done.
![Capture](https://user-images.githubusercontent.com/79115442/107994882-6899e280-7042-11eb-9300-b49bbe8d6d47.JPG)

**Comment by Scott-G** [comment]
Hard agree that NVMe support is needed but, even without it, this is still the best hardware monitor, by far.
